index.js

```diff
@@ -16,65 +20,67 @@ util.inherits(CpFileError, NestedError);
 
 CpFileError.prototype.name = 'CpFileError';
 
-module.exports = function (src, dest, opts, cb) {
+module.exports = function (src, dest, opts) {
 	if (!src || !dest) {
 		throw new CpFileError('`src` and `dest` required');
```

Comment:
This needs to be `Promise.reject(new CpFileError('`src` and `dest` required'));` as promise producing functions should never throw.